x-pack/plugins/search_playground/public/components/chat.tsx

```diff
@@ -115,6 +127,37 @@ export const Chat = () => {
 
               <EuiSpacer size="m" />
 
+              <EuiFlexGroup>
+                <EuiFlexItem grow={false}>
+                  <EuiButtonEmpty
+                    iconType="sparkles"
+                    disabled={chatMessages.length <= 1}
```

Comment:
Why it's not allowed to be 1 message?

Reply:
We are avoiding regenerate if there is no conversations happened. The first message is always the system message.